api/src/main/java/bio/terra/tanagra/service/query/api/ExpressionConverter.java

```diff
@@ -43,6 +43,10 @@
   }
 
   public Expression.Literal convert(ApiAttributeValue apiAttributeValue) {
+    if (apiAttributeValue == null) {
+      return Expression.Literal.create(DataType.STRING, null);
```

Comment:
Why STRING? Is this the default? maybe add a comment here

Reply:
The `DataType` here doesn't matter because when doing the special-case `null` handling, we just check whether the value is `null` and ignore the `DataType`. Added a comment.